### PR TITLE
Got ControlNetVersioner and related class to compile.

### PR DESCRIPTION
**Pull request by AgoinsUSGS**

### CODE REVIEW

**Comment by AgoinsUSGS** [comment]
[get ControlNetVersioner compiling](https://trello.com/c/2rMQMEct/139-get-controlnetversioner-compiling)

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -1358,7 +1364,7 @@ namespace Isis {
    *         file.
    */
   QSharedPointer<ControlMeasure> ControlNetVersioner::createMeasure(const ControlPointFileEntryV0002_Measure &measure) {
-    QSharedPointer<ControlMeasure> newMeasure = new QSharedPointer<ControlMeasure>();
+    QSharedPointer<ControlMeasure> newMeasure;
```
Comment (jessemapel) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: This is going to cause segfaults, it should use the QSharedPointer constructor:

QSharedPointer<ControlMeasure> newMeasuer(new ControlMeasure)

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 2]
```diff
@@ -1459,7 +1465,7 @@ namespace Isis {
 
       streampos startCoreHeaderPos = output.tellp();
 
-      OStreamOutputStream *fileStream(output);
+      OstreamOutputStream* fileStream = new OstreamOutputStream(&output);
```
Comment (jessemapel) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: This needs to be deleted at the end of the method

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 3]
```diff
@@ -1896,15 +1924,15 @@ namespace Isis {
           break;
 
         default:
-          IString msg = "The Pvl file version [" + IString(version) + "] is not"
+          QString msg = "The Pvl file version [" + QString(version) + "] is not"
```
Comment (jessemapel) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Change QString(version) to toString(version)

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Same for all of the other error messages

**Pull request merged**